test/triggerTest.fcl

```diff
@@ -55,7 +47,6 @@ services.TFileService.fileName : "test.root"
 physics.trigger_paths : [@sequence::Trig_physMenu.trigger_paths]
 
 #include "mu2e_trig_config/core/trigDigiInputsEpilog.fcl"
+#include "Production/Validation/database.fcl"
```

Comment:
I would prefer to avoid #include from Production. This is just a simple example to show how to run the trigger.